slack_bolt/function/Function.py

```diff
@@ -0,0 +1,60 @@
+from typing import List, Union, Pattern, Callable, Dict, Optional, Sequence, Any
+from logging import Logger
+
+from slack_bolt.listener_matcher import builtins as builtin_matchers
+
+from slack_bolt.response import BoltResponse
+from slack_bolt.middleware import Middleware
+
+
+# TDOD this is a duplicate function in App
+def _to_listener_functions(
+    kwargs: dict,
+) -> Optional[Sequence[Callable[..., Optional[BoltResponse]]]]:
+    if kwargs:
+        functions = [kwargs["ack"]]
+        for sub in kwargs["lazy"]:
+            functions.append(sub)
+        return functions
+    return None
+
+
+class Function:
+
+    def __init__(
+        self,
+        _register_listener: Callable[..., Optional[Callable[..., Optional[BoltResponse]]]],
```

Comment:
We can move the private function in App and use it here too. Let's remove this argument. This interface is not easy to use for developers.